static/app/views/replays/detail/page.tsx

```diff
@@ -33,12 +34,28 @@ function Page({children, crumbs, orgSlug, replayRecord}: Props) {
 
       <ButtonActionsWrapper>
         <ShareButton />
+        <FeatureFeedback featureName="replay" buttonProps={{size: 'sm'}} />
         <DeleteButton />
-        <FeatureFeedback featureName="replay" buttonProps={{size: 'xs'}} />
       </ButtonActionsWrapper>
 
       {replayRecord && crumbs ? (
-        <CrumbWalker replayRecord={replayRecord} crumbs={crumbs} />
+        <UserBadge
+          avatarSize={32}
+          displayName={<DisplayName>{replayRecord.user.display_name || ''}</DisplayName>}
+          user={{
+            username: replayRecord.user.display_name || '',
+            email: replayRecord.user.email || '',
+            id: replayRecord.user.id || '',
+            ip_address: replayRecord.user.ip || '',
+            name: replayRecord.user.username || '',
```

Comment:
Has this been fixed? I remember a support ticket where user was trying to search via username, but it was being aliased to `name` on the BE.

Reply:
i'll check what the backend has and get that fixed. 

We should have `user.username` on the ReplayRecord, per the docs: https://docs.sentry.io/product/sentry-basics/search/searchable-properties/

This UserBadge component accepts a few variations, i think we'll have to keep this mapping the way it is.